certutil/bundler.go

```diff
@@ -402,13 +402,13 @@ func constructCertFileName(cert *x509.Certificate) string {
 func (b *Bundler) fetchIntermediates(certs []*x509.Certificate) (err error) {
 	if IntermediateStash != "" {
 		if _, err := fileutil.Vfs.Stat(IntermediateStash); err != nil && os.IsNotExist(err) {
-			logger.Infof("intermediate stash directory %s doesn't exist, creating", IntermediateStash)
+			logger.KV(xlog.INFO, "intermediate stash directory doesn't exist, creating", IntermediateStash)
```

Comment:
`logger.KV(xlog.DEBUG, "reason", "intermediate stash directory not found", "folder", IntermediateStash)`

Reply:
I don't see this fixed